node/narwhal/src/bft.rs

```diff
@@ -706,7 +712,10 @@ mod tests {
         // Ensure this call fails on an even round.
         let result = bft.is_leader_quorum_or_nonleaders_available(2);
         assert!(result.is_err());
-        assert_eq!(result.unwrap_err().to_string(), "BFT storage reference is out of sync with the current round");
+        assert_eq!(
+            result.unwrap_err().to_string(),
+            "is_leader_quorum_or_nonleaders_available: BFT storage reference is out of sync with the current round"
```

Comment:
I wouldn't re-use the same error message - even in test, it's going to confuse people grepping for it...

Reply:
what do you suggest?

Reply:
I'd probably leave it unchanged - then the message is unique, and the test framework will show which test it's failing in...

Reply:
I don't understand what you mean. This text needs to match the actual error returned.

Reply:
Oh, sorry, then it's me who doesn't understand...